<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Regicide</string>
  </property>
  <widget class="QLabel" name="table">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(13, 145, 59, 255), stop:1 rgba(7, 102, 41, 255));
border:15px solid rgb(139, 0, 0);
border-radius: 60px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid rgb(0,0,0);
border-radius:12px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>410</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid rgb(255,255,255);
border-radius:12px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>59</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(200, 0, 0);</string>
   </property>
   <property name="text">
    <string>💖</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(200, 0, 0);</string>
   </property>
   <property name="text">
    <string>🗡</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>200</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid rgb(255,255,255);
border-radius:12px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>885</x>
     <y>380</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>墓地</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="remainCard">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>356</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
        border:3px solid black;
		border-radius:6px;
        color: white;
        text-align: center;
        background: rgb(68, 69, 73);
}
QProgressBar::chunk {

        background: rgb(0,210, 0);
}</string>
   </property>
   <property name="maximum">
    <number>52</number>
   </property>
   <property name="value">
    <number>9</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>887</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>酒馆</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="deathCard">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>566</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
        border:3px solid black;
		border-radius:6px;
        color:white;
        text-align: center;
        background: rgb(68, 69, 73);
}
QProgressBar::chunk {


        background: rgb(230,170, 0);
}
</string>
   </property>
   <property name="maximum">
    <number>52</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>592</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>手牌：</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="handbar_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>278</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
        border:3px solid black;
		border-radius:6px;
        color: white;
        text-align: center;
        background: rgb(68, 69, 73);
}
QProgressBar::chunk {

        background: rgb(28, 206, 255);
}</string>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string>%v/%m</string>
   </property>
  </widget>
  <widget class="QLabel" name="name3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>273</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>玩家4</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="handbar_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>388</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
        border:3px solid black;
		border-radius:6px;
        color: white;
        text-align: center;
        background: rgb(68, 69, 73);
}
QProgressBar::chunk {

        background: rgb(28, 206, 255);
}</string>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string>%v/%m</string>
   </property>
  </widget>
  <widget class="QLabel" name="name2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>383</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>玩家3</string>
   </property>
  </widget>
  <widget class="QLabel" name="name1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>493</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>玩家2</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="handbar_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>498</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
        border:3px solid black;
		border-radius:6px;
        color: white;
        text-align: center;
        background: rgb(68, 69, 73);
}
QProgressBar::chunk {

        background: rgb(28, 206, 255);
}</string>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string>%v/%m</string>
   </property>
  </widget>
  <widget class="QLabel" name="b">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid black;
border-radius:6px;
background-color:rgb(135, 135, 135)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="back2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid black;
border-radius:6px;
background-color:rgb(135, 135, 135)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="back1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid black;
border-radius:6px;
background-color:rgb(135, 135, 135)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="handCardBar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>594</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
        border:3px solid black;
		border-radius:6px;
        color: white;
        text-align: center;
        background: rgb(68, 69, 73);
}
QProgressBar::chunk {

        background: rgb(28, 206, 255);
}</string>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string>%v/%m</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>470</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>确认打出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="passButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>放弃回合</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>550</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="defendButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>弃牌</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 247);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: rgb(68, 69, 73);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="health">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>0</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: rgb(255,0,0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: rgb(68, 69, 73);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="att">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>0</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: rgb(150,150,150);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="heal_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>26</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="att_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Joker">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>40</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/image/image/Joker.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>table</zorder>
  <zorder>back1</zorder>
  <zorder>b</zorder>
  <zorder>back2</zorder>
  <zorder>label_2</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>remainCard</zorder>
  <zorder>label_15</zorder>
  <zorder>deathCard</zorder>
  <zorder>label_18</zorder>
  <zorder>handbar_4</zorder>
  <zorder>name3</zorder>
  <zorder>handbar_3</zorder>
  <zorder>name2</zorder>
  <zorder>name1</zorder>
  <zorder>handbar_2</zorder>
  <zorder>handCardBar</zorder>
  <zorder>confirmButton</zorder>
  <zorder>passButton</zorder>
  <zorder>sortButton</zorder>
  <zorder>defendButton</zorder>
  <zorder>label_3</zorder>
  <zorder>health</zorder>
  <zorder>label_4</zorder>
  <zorder>att</zorder>
  <zorder>heal_label</zorder>
  <zorder>att_label</zorder>
  <zorder>Joker</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
